Documentation/benchmarks/etcd-2-2-0-benchmarks.md

```diff
@@ -32,36 +32,36 @@ Bootstrap another machine and use benchmark tool [boom](https://github.com/rakyl
+| 256               | 256               | leader only        | 15239 (0%) | 28.4 (-3%) |
+| 64                | 64                | all servers        | 49502 (-3%) | 2.3 (+4%) |
+| 64                | 256               | all servers        | 50005 (-3%) | 9.1 (0%) |
+| 256               | 64                | all servers        | 48664 (-3%) | 2.3 (+4%) |
+| 256               | 256               | all servers        | 49914 (-4%) | 9.1 (-2%) |
 
 ### writing one single key
 
 | key size in bytes | number of clients | target etcd server | write QPS | 90th Percentile Latency (ms) |
 |-------------------|-------------------|--------------------|-----------|---------------|
-| 64                | 1                 | leader only        | 76 (+22%)  | 19.4 (-15%) |
-| 64                | 64                | leader only        | 2461 (+45%) | 31.8 (-32%) |
-| 64                | 256               | leader only        | 4275 (+1%) | 69.6 (-10%) |
-| 256               | 1                 | leader only        | 64 (+20%)  | 16.7 (-30%) |
-| 256               | 64                | leader only        | 2385 (+30%) | 31.5 (-19%) |
-| 256               | 256               | leader only        | 4353 (-3%) | 74.0 (+9%) |
-| 64                | 64                | all servers        | 2005 (+81%) | 49.8 (-55%) |
-| 64                | 256               | all servers        | 4868 (+35%) | 81.5 (-40%) |
-| 256               | 64                | all servers        | 1925 (+72%) | 47.7 (-59%) |
-| 256               | 256               | all servers        | 4975 (+36%) | 70.3 (-36%) |
+| 64                | 1                 | leader only        | 51 (-11%) | 22.3 (+1%) |
+| 64                | 64                | leader only        | 2182 (+14%) | 31.5 (-14%) |
+| 64                | 256               | leader only        | 5841 (+5%) | 52.0 (+2%) |
+| 256               | 1                 | leader only        | 57 (+7%) | 20.7 (-7%) |
+| 256               | 64                | leader only        | 1973 (+1%) | 38.3 (+8%) |
+| 256               | 256               | leader only        | 5193 (+4%) | 59.8 (-7%) |
+| 64                | 64                | all servers        | 1745 (+62%) | 54.2 (-49%) |
+| 64                | 256               | all servers        | 4487 (+21%) | 78.0 (-29%) |
+| 256               | 64                | all servers        | 1507 (+44%) | 59.5 (-43%) |
+| 256               | 256               | all servers        | 4231 (+15%) | 82.6 (-25%) |
 
 ### performance changes explanation
 
-- read QPS in most scenarios is decreased by 5~8%. The reason is that etcd records store metrics for each store operation. The metrics is important for monitoring and debugging, so this is acceptable.
+- read QPS in most scenarios is decreased by 3~7%. The reason is that etcd records store metrics for each store operation. The metrics is important for monitoring and debugging, so this is acceptable.
```

Comment:
How about:
"
Because etcd now records metrics for each API call, read QPS performance is decreased by approximately 3-7% in most scenarios. This minimal performance impact was judged a reasonable investment for the breadth of monitoring and debugging information returned.
"

Is that saying enough? -- or is it not sufficiently specific? There is a performance collection step on each API call, get or store, read or write?